commands/config.go

```diff
@@ -78,6 +78,31 @@ func getValue(path []string, obj interface{}) (interface{}, bool) {
 		return val.Interface(), true
 	} else if val.Kind() == reflect.Struct {
```

Comment:
`else` is not needed

Reply:
`mmctl config get PluginSettings.Plugins.com.mattermost.demo-plugin.channelname` will not panic, but will return:
`Error: Invalid key`

Reply:
why do you think that else is not needed here?

Reply:
It should cause error here `	val := r.FieldByName(path[0])`, because you have `map[string]map[string]interface{}` not `map[string]struct` and it has no field.